src/main/java/core/basesyntax/SupplyOperation.java

```diff
@@ -0,0 +1,15 @@
+import java.util.Map;
+
+public class SupplyOperation implements OperationHandler {
```

Comment:
The class declaration should not begin with an empty line. Please remove the empty line above the class declaration.